includes/Main.php

```diff
@@ -1405,16 +1447,25 @@ public function log_document_creation_trigger_to_order_meta( $document, $trigger
 	/**
 	 * Get the document triggers
 	 *
+	 * @param bool $only_manual_triggers
+	 *
 	 * @return array
 	 */
-	public function get_document_triggers() {
-		return apply_filters( 'wpo_wcpdf_document_triggers', [
+	public function get_document_triggers( bool $only_manual_triggers = false ): array {
+		$triggers = array(
```

Comment:
Consider improving this by separating the manual and automatic triggers, keeping them distinct and only combining them when necessary. Additionally, introducing separate filters for each group could enhance clarity and functionality.

Reply:
The function parameter could be enhanced by allowing it to accept values like 'manual', 'automatic', or 'all'.

Reply:
Since changing the filter could make Pro modifications not work, I didn't change it, but let me know if you have any ideas!

Reply:
Actually I was suggesting adding new filters for the groups, and not changing the one in the return.

Reply:
Got it, done!